frontend/src/components/datasets/DatasetsTableColumns.tsx

```diff
@@ -74,22 +75,31 @@ export const DATASET_COLUMN_DEFINITIONS : {[key in DatasetColumnID] : DatasetCol
 			return `${dataset.released_status_count}/${dataset.readset_count}`	// TODO display the total number of readsets
 		}
 	},
-	[DatasetColumnID.LATEST_UPDATE]: {
-		columnID: DatasetColumnID.LATEST_UPDATE,
+	[DatasetColumnID.LATEST_RELEASE_UPDATE]: {
+		columnID: DatasetColumnID.LATEST_RELEASE_UPDATE,
 		title: "Latest Release Status Update",
 		dataIndex: ['dataset', 'latest_release_update'],
 		render: (_, {dataset}) => {
 			return dataset.latest_release_update ? moment(dataset.latest_release_update).format("YYYY-MM-DD LT") : ""
 		}
-
+	},
+	[DatasetColumnID.LATEST_VALIDATION_UPDATE]: {
+		columnID: DatasetColumnID.LATEST_VALIDATION_UPDATE,
+		title: "Latest Validation Status Update",
+		dataIndex: ['dataset', 'latest_validation_update'],
+		render: (_, {dataset}) => {
+			return dataset.latest_validation_update ? moment(dataset.latest_validation_update).format("YYYY-MM-DD LT") : ""
```

Comment:
Should we keep using moment while it is not friendly with ant design? maybe move to dayjs or plain js date format ?

Reply:
I'm ok with leaving this moment since it's turned to a string before reaching antd. I can make removing moment part of it this PR if you'd like ;)

Reply:
nah i just thought we might stop using it in order to make it's removal easier later.

Reply:
I'd make it a separate PR to keep this PR simple and free of surprises.